package.json

```
@@ -529,6 +530,7 @@
   },
   "lint-staged": {
     "*.{js,jsx,ts,tsx}": [
+      "prettier --write",
       "eslint --ext '.js,.jsx,.ts,.tsx' -c .eslintrc.js"
```

wonder if we could just `--fix` since we already use `eslint-plugin-prettier`, plus this would also fix up other fixable eslint related issues while we're here

```suggestion
      "eslint --fix --ext '.js,.jsx,.ts,.tsx' -c .eslintrc.js"
```

---

Yeah fair. I don't know exactly, but I think most fixable issues in eslint are fixable by prettier as well, and prettier is many times faster. Atm I think the workflow is basically:
- `git commit`
- receive errors
- `yarn lint:js --fix` manually `yarn eslint` to pass in specific files
  - both of these are pretty slow

Basically I want the precommit hook to run as fast as possible, so prettier first and then eslint fits that bill. But maybe we should explore more! No rush on this, just noticed it was a little suss so figured I'd spend a few minutes on it.